# commnets include only ascii charcters.
Header header #std_msgs/Header
int8 status #express status example: 0:stanby 1:on 2:error
string text_ok
